.github/actions/setup-build/20-cadquery.yml

```
@@ -0,0 +1,38 @@
+name: "Setup CadQuery"
+description: "Prepare CadQuery and its dependencies"
+
+runs:
+  using: "composite"
+  steps:
+
+name: "Setup CadQuery"
```

YAMLlint (1.35.1)

[error] duplication of key "name" in mapping

(key-duplicates)